### PR TITLE
Replacement of vscode-nls with vscode.l10n

### PR DESCRIPTION
**Pull request by tehcrashxor**
Replacing `vscode-nls` with `vscode.l10n`

Note that this involves entirely removing the existing localization files for every other language, and we will need replacements from the Loc team before this is deployable.

[AB#3211613](https://dev.azure.com/dynamicscrm/1fb98997-2f9e-4734-be8a-9728680447c2/_workitems/edit/3211613)

### CODE REVIEW

**Comment by tehcrashxor** [comment]
Note: the current `loc\translations-import\vscode.powerplatform.fr.xlf` is a FAKE translation used to test the localization import and build process.   It needs to be removed, and a local build run to remove the generated fake translation artifacts, prior to completion of the PR.

**Review by petrochuk — APPROVED** [review]

**Review by swapnilbhalgat — APPROVED** [review]

**Review by tyaginidhi — APPROVED** [review]

File: loc/translations-import/vscode-powerplatform.cs.xlf [hunk 1]
```diff
@@ -1,462 +1,310 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.cs.xlf:1]: To keep Main clean for releases should one be needed prior to the new localization files, we'll keep this branch and PR unmerged until @toshio-msft and team have the recycled import XLFs pushed into this branch.

**Review by tehcrashxor — COMMENTED** [review]
Waiting to merge until the new import XLF files are available

File: loc/translations-import/vscode-powerplatform.cs.xlf (hunk 1 above)
Comment (tehcrashxor) [reply loc/translations-import/vscode-powerplatform.cs.xlf]: Got the import XLFs in, and tested the process with the supplied translations: 
✔️Spot checked Import XLF files
✔️build-time generation of Bundle and Package artifacts works as expected
✔️French strings displayed correctly on local Windows box (minus new strings which haven't been translated)
✔️Japanese and German strings displayed correctly GitHub CodeSpaces web client (minus new strings which haven't been translated)

**Pull request merged**